README.md

```diff
@@ -1,56 +1,20 @@
-# Benefits Eligibility - Phase 3
+# usagov-bears-spa
 
-BEARS will be a proactive notification service that provides timely and personalized information about benefits eligibility, contextualized 
```

Comment:
Did you mean to lose this in the Readme? Was it generated as part of the vue cli command or something?